src/pages/UpdateIncome.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { Fragment, useState } from 'react'
+import { Dialog, Transition } from '@headlessui/react'
+import {upDdateIncome} from '../utils/firebase-config'
+import { useAuth } from "../context/AuthContext";
+
+const UpdateIncome = ({showUpdateIncome,setShowUpdateIncome,setMonthlyIncome}) => {
+
+    const {currentUser} = useAuth();
+    const [newIncome, setNewIncome] = useState("") 
+
+    const addTransactions = (Income) => {
+        if(currentUser){
+            upDdateIncome(currentUser.uid,Income)
```

Comment:
is this line updating the income in the database?

Reply:
yeah, the line 14 is the update function